frontend/app/components/ToolCard.tsx

```diff
@@ -30,24 +27,7 @@ export default function ToolCard (props: ToolCardProps) {
   const { tool } = props;
 
   const [showMoreInfo, setShowMoreInfo] = useState(false);
-
-  let logoImageProps: CardImageProps = {
-    component: 'img',
-    height: 150,
-    alt: tool.logo_image_alt_text ?? 'Logo for tool'
-  };
-  if (tool.logo_image !== null) {
-    logoImageProps = { image: tool.logo_image, ...logoImageProps };
-  }
-
-  let mainImageProps: CardImageProps = {
-    component: 'img',
-    height: 150,
-    alt: tool.main_image_alt_text ?? 'Image of tool in use'
```

Comment:
Can the alt text contain tool name, such as "Screenshot of Gradecraft tool"

Reply:
Sure, I'll add that

Reply:
Should be changed now.